packages/snowpack/src/build/build-import-proxy.ts

```diff
@@ -3,6 +3,8 @@ import path from 'path';
 import {SnowpackConfig} from '../types/snowpack';
 import {getExt, URL_HAS_PROTOCOL_REGEX} from '../util';
 
+const CLOSING_BODY_TAG = /(<\s*\/\s*body\s*>)/i;
```

Comment:
Should we worry ourselves with an edge-case of multiple close body tags?

```html
<!-- </body> -->
</body>
```

Reply:
Comments… the bane of RegEx… Is there an easy fix to this, without stripping comments or parsing the HTML? If not, I’d like to avoid that rabbit hole. I’d hope that a comment with `</body>` in it would be pretty rare